eZ/Bundle/EzPublishCoreBundle/Resources/views/default/embed.html.twig

```diff
@@ -0,0 +1,3 @@
+{% for field in content.FieldsByLanguage %}
+    {{ ez_render_field(content, field.fieldDefIdentifier) }}
+{% endfor %}
```

Comment:
i think in legacy we had something called embed-full more similar to what's done here. 
For the intro summary the plan is provide those templates from this eZPublishCoreBundle?